### PR TITLE
fix: add logout as an explicit call

### PR DESCRIPTION
**Pull request by ivarconr**
I have not removed the LogoutComponent and the logout logic from the stores. Would we like to keep that around, or should I just remove it all?

### CODE REVIEW

File: src/component/menu/drawer.jsx [hunk 1]
```diff
@@ -99,6 +100,10 @@ export const DrawerMenu = ({
                         {item.title}
                     </NavLink>
                 ))}
+                <a className={classnames(styles.navigationLink)} href={`${getBasePath()}/logout`}>
+                    {getIcon('exit_to_app')}
```
Comment (FredrikOseberg) [inline src/component/menu/drawer.jsx:104]: Just merged a PR on the offline capabilities. With these changes, this needs to be changed to: 

```
import IconName from 'material-ui/core'

getIcon(IconName)
```

**Review by FredrikOseberg — CHANGES_REQUESTED** [review]
Looks good, due to some changes that were done in the meanwhile we need to change how we retrieve the icon.

**Comment by FredrikOseberg** [comment]
> I have not removed the LogoutComponent and the logout logic from the stores. Would we like to keep that around, or should I just remove it all?

I'd like to get rid of it, but we are currently using user.authDetails to determine whether or not a user is authenticated in the frontend. If we no longer update this on logout, we need another method of determining user authentication.

**Comment by ivarconr** [comment]
> If we no longer update this on logout, we need another method of determining user authentication.

we kind of do because we instructs the browser to clear all cookies and local storage, so the authDetails will be gone after the server-side redirect.

File: src/component/menu/__tests__/routes-test.jsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { baseRoutes, getRoute } from '../routes';
 
 test('returns all baseRoutes', () => {
-    expect(baseRoutes.length).toEqual(12);
+    expect(baseRoutes.length).toEqual(11);
```
Comment (chriswk) [inline src/component/menu/__tests__/routes-test.jsx:4]:
```suggestion
    expect(baseRoutes).toHaveLength(11);
```

**Pull request merged**